### PR TITLE
Refactor octobot py

### PR DESCRIPTION
**Pull request by Werdor**
Refactor octobot.py splitted in 4 new files : 
- initializer.py
- exchange_factory.py
- evaluator_factory.py
- task_manager.py

### CODE REVIEW

File: core/initializer.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+from evaluator.Util import AdvancedManager
+from evaluator.evaluator_creator import EvaluatorCreator
+from services import ServiceCreator
+from tools import get_logger
+from tools.notifications import Notification
+from tools.performance_analyser import PerformanceAnalyser
+from tools.time_frame_manager import TimeFrameManager
+
+
+class Initializer:
```
Comment (GuillaumeDSM) [inline core/initializer.py]: I think it would be easier to read with _ prefixes for protected methods

Comment (GuillaumeDSM) [reply core/initializer.py]: Good idea to wrap initalizations in one file :)

File: setup.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#  Drakkar-Software OctoBot-Launcher
```
Comment (GuillaumeDSM) [inline setup.py]: 👍

File: core/initializer.py (hunk 1 above)
Comment (Werdor) [reply core/initializer.py]: Yes good idea ;)

**Pull request merged**